spec/order_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'order'
+describe Order do 
+  let (:pasta){double :dish, name: 'pasta', price: 6}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.